libs/domains/clusters/feature/src/lib/cluster-type/cluster-type.tsx

```diff
@@ -17,7 +17,7 @@ export function ClusterType({ cloudProvider, kubernetes, ...props }: ClusterType
     .with(['SCW', P._], () => 'Managed (Kapsule)')
     .exhaustive()
   return (
-    <Badge color="neutral" {...props}>
+    <Badge size="xs" color="neutral" {...props}>
```

Comment:
Should be on `ClusterType` usage not here

Reply:
https://github.com/Qovery/console/pull/851/commits/630de3667e7445b8f9b76094ecc896b547a008d6